File: atcoder/abc146A_cant_wait_for_holiday.hs
-- unihernandez22
-- https://atcoder.jp/contests/abc146/tasks/abc146_a
-- implementation

ans li | li == "SUN" = 7
       | li == "MON" = 6
       | li == "TUE" = 5
       | li == "WED" = 4
       | li == "THU" = 3
       | li == "FRI" = 2
       | otherwise   = 1


main = do
  li <- getLine
  putStrLn $ show $ ans li

File: atcoder/abc146A_cant_wait_for_holiday.ml
(*(* vicfred
 * https://atcoder.jp/contests/abc146/tasks/abc146_a
 * implementation
 * *)
Printf.printf "%d\n" @@ match read_line () with
| "SUN" -> 7
| "MON" -> 6
| "TUE" -> 5
| "WED" -> 4
| "THU" -> 3
| "FRI" -> 2
| "SAT" -> 1
| _ -> -1

